save-backend/backend-api-docs.json

```diff
@@ -8541,13 +8541,13 @@
               "DECEMBER"
             ]
           },
-          "monthNumber": {
```

Comment:
it's changed from time to time without any changes in endpoints: @sanyavertolet, @petertrr do you have any ideas how to avoid it?

Reply:
Will experiment in other PR